src/models/multiBarChart.js

```diff
@@ -334,12 +334,14 @@ nv.models.multiBarChart = function() {
 
     multibar.dispatch.on('elementMouseover.tooltip', function(evt) {
         evt.value = chart.x()(evt.data);
-        evt['series'] = {
-            key: evt.data.key,
-            value: chart.y()(evt.data),
-            color: evt.color
-        };
-        tooltip.data(evt).hidden(false);
+        if (!chart.stacked() || (evt.data.size > 0 || evt.data.size == undefined)) {
```

Comment:
I'm pretty sure I have it. Look I'm now using the same d3.js version you put in the jsfiddle and nvd3 version 1.8.1, and I have the `size` in every multiBarChart (stacked).